examples/docs/es/date-picker.md

```diff
@@ -92,14 +92,15 @@
 </style>
 
 ## DatePicker
+Utiliza Date Picker para introducir fecha.
 
-Use Date Picker for date input.
+###  Entrar fecha
 
-###  Enter Date
+Selector de fecha básico limitado por día.
 
-Basic date picker measured by 'day'.
+:::manifestación Ésta medición es decidido por el atributo 'type'.Puedes permitir rápidas opcíones por crear un `picker-options` objecto con propiedad `shortcuts`.La fecha minusválida es establecido por 
```

Comment:
```
:::demo
```

```
:::tip
```
```
:::warning
```

These are symbols for vue-markdown-loader to parse, so please don't translate them.